data/korath/korath outfits.txt

```diff
@@ -91,6 +91,32 @@ outfit "Large Heat Shunt"
 	"cooling" 54
 	description "Without these powerful cooling systems, most Korath warships would overheat within seconds as soon as they began to fire their weapons."
 
+outfit "Liquid Sodium Cooler"
+	category "Systems"
+	cost 1660000
+	thumbnail "outfit/liquid sodium"
+	"mass" 37
+	"outfit space" -34
```

Comment:
I don't really mind as long as there is some reason for it.

Reply:
Well, the reason I like to do these things is because, when left unexplained, it makes the player go "Oh, whoa, these high-tech outfits are weird," creating a sense of wonder in a very minor way.